chirpus-2/index.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400&family=Roboto&display=swap');
+body {
+    margin: 10px;
+    background-color: #f9fafc;
+    font-family: 'Open-Sans', sans-serif;
+}
+.nav {
+    display: flex;
+    position: relative;
+    flex-direction: column;
+    color: #69696a;
+    padding: 20px;
+}
+.nav__logo {
+    position: absolute;
+    padding: 5px;
+    left: 10px;
+    color: #000;
+}
+.nav__links__link {
+    display: flex;
+    justify-content: flex-end;
```

Comment:
I think you wanted this justify-content: flex-end inside of your media query. This is sending your links to the right side of the page when in mobile view, but we only want them on the right in desktop view.